pkg/networkmanager/ip-settings.jsx

```diff
@@ -271,7 +271,7 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
                                 <Switch
                                     isChecked={!ignoreAutoDns}
                                     isDisabled={!canAuto}
-                                    onChange={value => setIgnoreAutoDns(!value)}
+                                    onChange={(_event, value) => setIgnoreAutoDns(!value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-161212-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)